reports/performance/_outliers.qmd

```diff
@@ -368,3 +368,34 @@ qnt_df <- assessment_pin %>%
 qnt_df %>%
   datatable(rownames = FALSE)
 ```
+## Map of Outliers
+
+This map identifies the ratio of sale price to the initial fair market value. It filters the the data to where our estimate is abnormally high (>99th percentile) and abnormally low (<1st percentile).
+```{r outlier_map, message=FALSE}
+outlier_dist <- test_card %>%
+  group_by(meta_township_code) %>%
+  mutate(
+    ratio = meta_sale_price / pred_card_initial_fmv,
+    quantile_category = case_when(
+      ratio <= quantiles$q01 ~ "Abnormally High Valuation",
+      ratio >= quantiles$q99 ~ "Abnormally Low Valuation",
```

Comment:
**issue:** This throws the following error when I try to render your changes:

```
Quitting from lines 375-401 [outlier_map] (performance.qmd)
Error in `mutate()`:
ℹ In argument: `quantile_category = case_when(...)`.
ℹ In group 1: `meta_township_code = "10"`.
Caused by error in `case_when()`:
! Failed to evaluate the left-hand side of formula 1.
Caused by error:
! object 'quantiles' not found
```

Reply:
Sorry, this was a silly mistake where I tried to condense the code into one chunk, but since I had things stored locally, it rendered when running the chunk. I reverted to the previous version and it works fine.